languages/csharp/exercises/concept/strings/.docs/introduction.md

```diff
@@ -1,3 +1,4 @@
 # Introduction
 
-A `string` in C# is a sequence of Unicode characters (letters, digits, punctuation, etc.).
+A `string` in C# is an object that holds and manipulates an arbitrary sequence of bytes, typically representing characters. Strings are manipulated by calling the string's methods.
```

Comment:
```suggestion
A `string` in C# is an object that holds a sequence of Unicode characters (letters, digits, punctuation, etc.). Strings are manipulated by calling the string's methods. Once a string has been constructed, its value can never change. Any methods that appear to modify a string will actually return a new string.
```

Reply:
@EarthlingRich What do you think of the above suggestion?

Reply:
I would think it is worth giving one or two examples - reminding the traveller from Javascript or elsewhere that double quotes are mandatory and how we do verbatim strings in our language and maybe slip in an escaped character and a unicode code or two to show them how it's done.

Reply:
That is a great description @ErikSchierboom. The suggestions of @mikedamay are out of scope for this PR. It's more about the exercise itself.

Reply:
> I would think it is worth giving one or two examples - reminding the traveller from Javascript or elsewhere that double quotes are mandatory and how we do verbatim strings in our language and maybe slip in an escaped character and a unicode code or two to show them how it's done.

@mikedamay The `instructions.md` document's aim is to provide a minimal description. The idea is that the student will figure these things out by themselves, as long as we'll provide them with the minimal context (which is why the text mentions `string` as the type).

@EarthlingRich Do you perhaps have a suggestion?

Reply:
@EarthlingRich I've updated

Reply:
Great! 👍